examples/solids/README.rst

```diff
@@ -64,7 +60,7 @@ Consider the specific example of the mesh seen below:
 
 With the sidesets defined in the figure, we provide here an example of a minimal set of command line options::
 
-   ./elasticity -mesh [.exo file] -degree 4 -E 1e6 -nu 0.3 -bc_zero 999 -bc_clamp 998 -bc_clamp_998_translate 0,-0.5,1
+   ./elasticity -mesh [.exo file] -degree 4 -E 1e6 -nu 0.3 -bc_clamp 998,999 -bc_clamp_998_translate 0,-0.5,1
```

Comment:
Thanks. Perhaps you want to update the following line, too?

Reply:
I'm not sure if its clear, but here I mean to be reinforcing that the option `-bc_clamp` without `-bc_clamp_[facenum]_[translate/rotate]`  sets as default displacement of zero displacement on that face.

Reply:
You can remove the reference to `-bc_zero` in `elasticity.c`.

Reply:
Got it

Reply:
Yes the part for which `-bc_clamp` without `[facenum]_[translate/rotate]` sets the default zero displacement condition on the face specified was clear. What was not entirely clear to me is if `-bc_clamp 998,999`  specifies this condition on both facesets labeled as 998 and 999. And then in the following sentence (on which I could not comment in the review, because was not modified) it reads "In this example, we set the left boundary, face set :math:`999`, to zero displacement"

Reply:
`-bc_clamp_[facenum]_[translate/rotate]` always needs to be preceded by `-bc_clamp [int list that contains facenum]` so that I can build the strings for the command line options corresponding to the translation and/or rotation vectors on that face.

Reply:
Correct.

Reply:
Ok, if one option must necessarily precede the other, it is something that we should make sure it's specified somewhere either in the documentation or as a comment right after the sample runs in the `elasticity.c` file. Otherwise a user may accidentally use them in arbitrary order. 

The following line here could become
```suggestion
   In this example, we set the boundary conditions for the displacement field on the face sets denoted by math:`:998,999`. We set the right boundary, face set :math:`998`, to translate :math:`0` in the :math:`x` direction, :math:`-0.5` in the :math:`y`, and :math:`1` in the :math:`z`, and the left boundary, face set :math:`999`, with unspecified condition, is defaulted to to zero displacement.
```

As I said on Zulip, to me personally it would sound clearer if these were expressed as `-bc_displacement 998,999` or `-bc_disp 998,999` for short, and then `-bc_disp_998_translate 0,-0.5,1`. But it's totally up to you of course.

Reply:
Sorry my email notifications are lagged. I just received the one about the merge just now.